code/modules/clothing/spacesuits/ert_hardsuits.dm

```diff
@@ -1,7 +1,7 @@
 
 /obj/item/clothing/head/helmet/space/hardsuit/ert
 	name = "emergency response team helmet"
-	desc = "A helmet worn by members of the Nanotrasen Emergency Response Team. Armoured and space ready."
+	desc = "An environmentally sealed combat helmet with a wide plexiglass visor for maximum visibility. The HUD's software licensce has expired."
```

Comment:
Do we want to imply that NT ERT's are poorly funded? The expired HUD license certainly implies that.